scripts/validation.js

```diff
@@ -0,0 +1,23 @@
+export const email = document.getElementById('email');
+export const formRef = document.querySelector('form');
+
+const errorMsg = document.getElementById('myPopup');
+const subBtn = document.getElementById('sub-btn');
+const messages = ['No caps allowed!!', 'Write a valid email please!', 'Remeber to use a valid domain!!',
+  'NO CAAAPS!!', 'Maybe you are missing .com?', 'Please use lower case letters.', 'We cannot accept the caps in your email.'];
+
+const pickRandomMessage = () => messages[Math.floor(Math.random() * messages.length)];
+
+export const isValidMail = () => {
+  if (/[A-Z]/g.test(email.value) || email.validity.typeMismatch) {
+    subBtn.classList.add('disable');
+    errorMsg.classList.add('show');
+    errorMsg.innerText = pickRandomMessage();
```

Comment:
- [ ] **As you are choosing the error message randomly, it may necessarily not inform about the issue with an uppercase email. Please refer to the image below.**

![image](https://user-images.githubusercontent.com/59168430/123340591-0e69da80-d56a-11eb-898a-5d2bd1955fa1.png)

Please make sure that the error message is specifically about the case sensitivity of email if the user enters a capitalized email.